.changeset/gorgeous-keys-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware-pwa/nuxt3-module": minor
+---
+
+Add private `shopwareEndpoint` config for SSR instance creation
```

Comment:
we need some docs how to use it and where it is necessary

Reply:
And also explain when to use what ... because we also have this one:
https://frontends.shopware.com/packages/api-client/docs/storefront-url.html#preview-instance